templates/out-of-sync.j2

```diff
@@ -77,7 +77,7 @@ $(document).ready(function () {
         <th>Verified at</th>
         <th>Mathlib3 changes (newest first)</th>
         <th>&pm;</th>
-        <th>import order</th>
+        <th>import depth</th>
```

Comment:
```suggestion
        <th title="...">import depth</th>
```
where ... is some explanation of what this number means (longest chain of dependencies that import this file?)